docs/dev-guide/adding-grid-support/adding-grid-support-step-by-step-guide/add-grid-config.md

```diff
@@ -1,3 +1,175 @@
 # Add New Grid Configuration to E3SM
 
+In addition to generating input data to support a new grid, several code modifications are required before E3SM can run with the grid. However, the specific changes will depend on how the grid will be used. The intendend model configuration for the new grid will change which files need to be modified. For instance, a grid intended for aquaplanet experiments does not require as many changes as a historical AMIP-style run.
+
+The guidelines here are meant to outline various possible changes the user should consider when adding support for a new grid. This document cannot be exhaustive, and it is important that the user understands the changes they are making. It is often useful to use a pre-existing grid configuration as a template. Note that the guidelines here are only relevant for "horizontal" grids. Similar considerations are needed to support a new vertical grid.
+
+When setting up a new grid you will need to edit some or all of these files:
+
+- `cime_config/config_grids.xml`
+- `components/eam/bld/config_files/horiz_grid.xml`
+- `components/eam/bld/namelist_files/namelist_defaults_eam.xml`
+- `components/eam/bld/namelist_files/namelist_definition.xml`
+- `components/elm/bld/namelist_files/namelist_definition.xml`
+- `components/elm/bld/namelist_files/namelist_defaults.xml`
+
+## Mono-Grid vs Bi-Grid vs Tri-Grid
+
+The mono-bi-tri grid options in E3SM can be confusing, but it's important to understand what these terms mean when adding a new grid to E3SM. At the surface these terms mean that the whole model either using a single grid for all componennt models, or a combination of 2 or 3 grids shared among the component models. Note that mono-grid and bi-grid terms often ignore that the river model needs to be on i/ts own regular lat-lon grid (for now).
```

Comment:
Also, I think "i/ts" should be "its".

Reply:
good catches

Reply:
actually "it's" is correct here because you only use the apostrophe for the contraction of "it is", whereas the possessive form is "its" - but I'll just write out "it is"